examples/node/pdf2svg.js

```diff
@@ -87,56 +87,36 @@ function writeSvgToFile(svgElement, filePath) {
   });
 }
 
-// Will be using promises to load document, pages and misc data instead of
-// callback.
+// Will be using async/await to load document, pages and misc data.
 const loadingTask = pdfjsLib.getDocument({
   data,
   cMapUrl: CMAP_URL,
   cMapPacked: CMAP_PACKED,
   fontExtraProperties: true,
 });
-loadingTask.promise
-  .then(function (doc) {
-    const numPages = doc.numPages;
-    console.log("# Document Loaded");
-    console.log("Number of Pages: " + numPages);
-    console.log();
+(async function () {
+  const doc = await loadingTask.promise;
+  const numPages = doc.numPages;
+  console.log("# Document Loaded");
+  console.log(`Number of Pages: ${numPages}`);
+  console.log();
 
-    let lastPromise = Promise.resolve(); // will be used to chain promises
-    const loadPage = function (pageNum) {
-      return doc.getPage(pageNum).then(function (page) {
-        console.log("# Page " + pageNum);
-        const viewport = page.getViewport({ scale: 1.0 });
-        console.log("Size: " + viewport.width + "x" + viewport.height);
-        console.log();
+  for (let pageNum = 1; pageNum <= numPages; pageNum++) {
+    try {
+      const page = await doc.getPage(pageNum);
+      console.log(`# Page ${pageNum}`);
+      const viewport = page.getViewport({ scale: 1.0 });
+      console.log(`Size: ${viewport.width} x ${viewport.height}`);
```

Comment:
This is introducing adding additional spaces in the string, why?
The following is equivalent to the old code:
```javascript
console.log(`Size: ${viewport.width}x${viewport.height}`);
```

Reply:
@Snuffleupagus Sorry, I missed this. It has been fixed.